app/views/links/edit.js.erb

```diff
@@ -1 +1,2 @@
-$('#link-form').html("<%= j (render 'edit_form') %>").slideDown(350);
+$('.modal-body').html("<%= j(render 'edit_form') %>");
+$('#edit-modal').modal();
```

Comment:
**UX-Critical**
Incorporate bootstrap modal for better responsiveness with smaller screens.